documentation/4.0/content/release-notes.md

```diff
@@ -10,6 +10,7 @@ draft: false
 
 | Date               | Version  | Change - See also, [Change log](#change-log).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 |--------------------|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| February 21, 2023  | v4.0.5   | Resolved.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
A summary of resolved issues is missing. Maybe this is intentional, but just want to make sure.

Reply:
Thanks! Yes, you're right, this was supposed to be a "todo" marker so that I would write this message.